src/main/java/io/supertokens/Main.java

```diff
@@ -188,7 +188,7 @@ private void init() throws IOException, StorageQueryException {
             }
         }
         try {
-            StorageLayer.getBaseStorage(this).initStorage(true, List.of(TenantIdentifier.BASE_TENANT));
+            StorageLayer.getBaseStorage(this).initStorage(true, List.of());
```

Comment:
Are you sure we should not pass in base_tenant here? If for any reason public is not there in the tenants table, will that cause an issue?